### PR TITLE
Add UILink.

### PR DESCRIPTION
**Pull request by SAnatoliiS**

### CODE REVIEW

**Comment by SAnatoliiS** [comment]
image
image

File: casts/row/src/UILink/types.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import type { ImageSourcePropType } from 'react-native';
+
+export type UILinkProps = {
+    title?: string;
+    description?: string;
+    logoSource?: ImageSourcePropType;
+    iconAfterTitleSource?: ImageSourcePropType;
```
Comment (savelichalex) [inline casts/row/src/UILink/types.ts]: I don't think it should be configurable. Let's drop this prop

File: casts/row/src/UILink/types.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import type { ImageSourcePropType } from 'react-native';
+
+export type UILinkProps = {
+    title?: string;
+    description?: string;
+    logoSource?: ImageSourcePropType;
```
Comment (savelichalex) [inline casts/row/src/UILink/types.ts]: What do you think to call it just `logo`?

File: casts/row/src/UILink/UILink.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    title,
+    description,
+    logoSource,
+    iconAfterTitleSource,
+    onPress,
+    loading,
+    testID,
+}: UILinkProps) => {
+    return (
+        <UISkeleton show={!!loading}>
```
Comment (savelichalex) [inline casts/row/src/UILink/UILink.tsx]: I actually thought that skeleton will be around logo, title and subtitle, not around whole row. Like the point of skeletons is to show what layout will be after it's loaded

Comment (SAnatoliiS) [reply casts/row/src/UILink/UILink.tsx]: I have separated the logo from the text block
image

**Review by savelichalex — APPROVED** [review]

File: casts/row/src/UILink/Logo.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import type { LogoProps } from './types';
+import { UIConstant } from '../constants';
+
+export const Logo: React.FC<LogoProps> = ({ logo, loading }: LogoProps) => {
+    if (!logo) {
+        return (
+            <View style={styles.container}>
+                <UISkeleton show={!!loading}>
+                    <View style={styles.image} />
+                </UISkeleton>
```
Comment (sertony) [inline casts/row/src/UILink/Logo.tsx]: Am I right that the skeleton will look more like a square here than a plain circle (as the logo should look like)?

Comment (savelichalex) [reply casts/row/src/UILink/Logo.tsx]: A logo isn't a circle anymore :)

Comment (SAnatoliiS) [reply casts/row/src/UILink/Logo.tsx]: image

**Review by sertony — APPROVED** [review]

**Pull request merged**